src/components/index/why-netlogo.tsx

```diff
@@ -44,21 +46,18 @@ const WhyNetLogo = () => {
         moreButton={false}
         body={
           <div className="why-netlogo-content">
-            <ForTab
-              title="For Students"
-              content="NetLogo provides students with many pre-made models of scientific phenomena they can explore. For those who want to learn to program or create models themselves, NetLogo is very easy to get started with."
-              imagePath={student}
-            />
-            <ForTab
-              title="For Researchers"
-              content="NetLogo is easy to learn but still very powerful. It is has been used in over 20,000 scientific publications in fields including sociology, ecology, cognitive science, business, and more."
-              imagePath={researcher}
-            />
-            <ForTab
-              title="For Educators"
-              content="NetLogo provides educators with an easy-to-use modeling platform that includes many built-in models to engage students in learning science."
-              imagePath={educator}
-            />
+            {page_data?.length > 0 ? (
+              page_data.map((entry) => (
+                <ForCard
+                  key={entry.id}
+                  title={entry.title}
+                  content={entry.content}
+                  imageKey={entry.icon}
+                />
+              ))
+            ) : (
+              <p>Loading or no data available...</p>
```

Comment:
You don't have this `<p>Loading or no data available...</p>` anywhere else. Why here? Isn't this part of the page supposed to be pre-built and served statically?